users/vieting/experiments/switchboard/ctc/feat/network_helpers/perturbation.py

```diff
@@ -121,17 +123,21 @@ def preemphasis_numpy(waveform, coeff=0.97):
 
     @staticmethod
     def apply_codecs(audio, sample_rate, random_state, codecs):
-        import sox
-
-        tfm = sox.Transformer()
         for codec in codecs:
             prob = codec.pop("prob", 1.0)
             if random_state.random() < prob:
                 if codec.get("encoding") == "ULAW":
-                    tfm.set_output_format(encoding="u-law")
+                    # standard value for mu-law encoding
+                    mu = 255.0
+
+                    # mu-law encoding formula
```

Comment:
Didn't you previously check that the audio signal is in the range [-1, 1]?

Reply:
added back in, but it should not be necessary as Switchboard is loaded in the correct format.